### ISSUE TITLE
Stylesheets repeat

### ISSUE THREAD

**Issue by sarahall** [CityOfPhiladelphia/parks-rec-finder#121]
Remove excess stylesheets

**Comment by karissademi** [comment]
This is likely due to some issue with the build process.

**Comment by timwis** [comment]
The only repeats I see in the production build are 4 style tags for `.esri-truncated-attribution`, though they only have one CSS rule in them, so not that bad. What are you seeing? Is it prod or dev?

**Comment by karissademi** [comment]
You're right, the issue is with dev, not prod. It could be nodejs version conflicting with a package or something about the config. Regardless, dev is not a high priority at the moment. 

At first glance I thought it was happening in prod too, but it's really just that there are numerous overrides of styles that bulk up the inspector. It might be a good idea to do a style audit to verify that all the dependencies are needed.
